src/workloads/execution/PipelineUpdate.yml

```diff
@@ -47,193 +47,244 @@ Actors:
   - *Nop
   - *Nop
   - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2, 4, 6, 8]
+      NopInPhasesUpTo: 9
+      PhaseConfig:
+        Repeat: 1
 
 - Name: RunUpdates
   Type: RunCommand
   Threads: 16
   Phases:
   - *Nop
   - *Nop
+  - *Nop
   - Repeat: 500    # Classic updates on small documents phase.
```

Comment:
[suggestion] Feel free to ignore to limit the changes, but:

```
  - Repeat: &RunCommandRepeat 500    # Classic updates on small documents phase.
Write
```
And then later:
```
  - Repeat: *RunCommandRepeat
```

If having all the same Repeat is desirable.

Reply:
Good idea, I've made the corresponding change.